apps/core/lib/core/tools/helpers/query_helper/query_helper.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Core.Tools.Helpers.QueryHelper do
+defmodule Snitch.Core.Tools.Helpers.QueryHelper do
```

Comment:
Why does this namespace have `Helpers`? Choose between:
1. `Snitch.Core.Tools.Helpers.Query`
2. `Snitch.Core.Tools.QueryHelper` _(preferred as it is shorter)_